ros_control_loop_frequency:              100.0
publish_learning_mode_frequency:         2.0


